src/SearchQueryService/Indexes/Models.cs

```diff
@@ -0,0 +1,101 @@
+﻿using MMLib.ToString.Abstraction;
```

Comment:
Dodrž prosím .NET konvencie (čo trieda to súbor)

Taktiež je nedodržaných viacero konvencií (prázdne riadky medzi properties, ...) Keď pridáš vyššie spomínaný `.editorconfig` a spustíš build tak by si mal dostať warninngy, odstráň ich 😉.